iib/workers/tasks/build.py

```diff
@@ -929,17 +934,22 @@ def handle_rm_request(
     with tempfile.TemporaryDirectory(prefix='iib-') as temp_dir:
         with set_registry_token(overwrite_from_index_token, from_index_resolved):
             if is_image_fbc(from_index_resolved):
-                err_msg = 'File-Based catalog image type is not supported yet.'
-                log.error(err_msg)
-                raise IIBError(err_msg)
+                log.info("Processing File-Based Catalog image")
+                opm_registry_rm_fbc(
+                    base_dir=temp_dir,
+                    from_index=from_index_resolved,
+                    operators=operators,
+                    binary_image=prebuild_info['binary_image'],
+                )
```

Comment:
Is this related to `create-empty-index`?

Because it's touching `handle_rm_request` and I would like to know whether is a requirement for `cei` it or not.

Reply:
this is from another commit related to remove operator, once  #[342] is merged and I will rebase this over that

Reply:
Ok, I've added comments over the places I found that, hope it's useful.

Once it's rebased just let me know then I review it again